Doc/library/enum.rst

```diff
@@ -847,7 +847,7 @@ Utilities and Decorators
 .. decorator:: unique
 
    A :keyword:`class` decorator specifically for enumerations.  It searches an
-   enumeration's :attr:`__members__`, gathering any aliases it finds; if any are
+   enumeration's :attr:`!EnumType.__members__`, gathering any aliases it finds; if any are
```

Comment:
Same here. Either ``__members__`` should be documented (now or in the immediate future), and this should be:

```suggestion
   enumeration's :attr:`~EnumType.__members__`, gathering any aliases it finds; if any are
```

Or, since this is again the word after the possisive "enumeration" indicating to what it belongs, if it shouldn't be documented, it should just be left as

```suggestion
   enumeration's :attr:`!__members__`, gathering any aliases it finds; if any are
```